### PR TITLE
Separate match snapshots from visual tests

### PR DESCRIPTION
**Pull request by mharsat**

### CODE REVIEW

File: .github/workflows/actions.yml [hunk 1]
```diff
@@ -19,6 +19,22 @@ jobs:
       APPLITOOLS_API_KEY: ${{ secrets.APPLITOOLS_API_KEY }}
       APPLITOOLS_SERVER_URL: ${{ secrets.APPLITOOLS_SERVER_URL }}
       APPLITOOLS_BATCH_ID: ${{ github.sha }}
+  e2e-content-tests:
```
Comment (shaulgo) [inline .github/workflows/actions.yml]:
```suggestion
  e2e-content-state-tests:
```

File: .github/workflows/actions.yml [hunk 2]
```diff
@@ -19,6 +19,22 @@ jobs:
...
+        uses: actions/checkout@v1
+
+      - name: Cypress run
+        uses: cypress-io/github-action@v1
+        with:
+          build: npm run build:test
+          start: npm run e2e:app:run
+          wait-on: 'http://localhost:3002'
+          config: video=false
+          env: MATCH_CONTENT=true
```
Comment (shaulgo) [inline .github/workflows/actions.yml]:
```suggestion
          env: MATCH_CONTENT_STATE=true
```

File: e2e/cypress/support/commands.js [hunk 3]
```diff
@@ -61,9 +61,10 @@ Cypress.Commands.add('loadEditorAndViewer', fixtureName => {
 });
 
 Cypress.Commands.add('matchContentSnapshot', () => {
-  cy.window()
-    .its('__CONTENT_SNAPSHOT__')
-    .toMatchSnapshot();
+  if (Cypress.env('MATCH_CONTENT'))
```
Comment (shaulgo) [inline e2e/cypress/support/commands.js]:
```suggestion
  if (Cypress.env('MATCH_CONTENT_STATE'))
```

File: package.json [hunk 4]
```diff
@@ -22,8 +22,10 @@
     "createVersion": "node scripts/createVersion",
     "e2e": "start-server-and-test e2e:app http://localhost:3002 e2e:run",
     "e2e:debug": "npm run e2e:app:dev & cypress open",
+    "e2e:testContent": "start-server-and-test e2e:app http://localhost:3002 e2e:run:testContent",
     "e2e:updateSnapshots": "start-server-and-test e2e:app http://localhost:3002 e2e:run:update",
     "e2e:run": "npx cypress run",
+    "e2e:run:testContent": "APPLITOOLS_IS_DISABLED=true npx cypress run --env MATCH_CONTENT=true",
```
Comment (shaulgo) [inline package.json]:
```suggestion
    "e2e:run:testContent": "APPLITOOLS_IS_DISABLED=true npx cypress run --env MATCH_CONTENT_STATE=true",
```

File: e2e/tests/rendering.e2e.js [hunk 5]
```diff
@@ -17,6 +17,10 @@ const testFixture = fixture =>
   });
 
 describe('editor rendering', () => {
+  before(function() {
```
Comment (shaulgo) [inline e2e/tests/rendering.e2e.js]: should move above this line? So it includes also the mobile.

Comment (mharsat) [reply e2e/tests/rendering.e2e.js]: using this syntax it does include both contexts

File: e2e/tests/rendering.e2e.js [hunk 6]
```diff
@@ -17,6 +17,10 @@ const testFixture = fixture =>
...
+    if (Cypress.env('MATCH_CONTENT')) this.skip();
```
Comment (shaulgo) [inline e2e/tests/rendering.e2e.js]:
```suggestion
    if (Cypress.env('MATCH_CONTENT_STATE')) this.skip();
```

**Pull request merged**